packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/index.js

```diff
@@ -69,15 +71,65 @@ const confirmDialogsPropTypes = {
  * ConfirmDialogPublishAll
  * -----------------------------------------------------------------------------------------------*/
 
-const ConfirmDialogPublishAll = ({ isOpen, onToggleDialog, isConfirmButtonLoading, onConfirm }) => {
+const ConfirmDialogPublishAll = ({
+  isOpen,
+  onToggleDialog,
+  isConfirmButtonLoading,
+  onConfirm,
+  slug,
+}) => {
   const { formatMessage } = useIntl();
+  const { get } = useFetchClient();
+  const { selectedEntries } = useTableContext();
+
+  const { data: countDraftRelations } = useQuery(
+    ['content-manager', 'draft-relations'],
+    async () => {
+      try {
+        const {
+          data: { data },
+        } = await get(
```

Comment:
Could you please add a TODO here, to refactor this once the param serializer work from Mark is merged?

Reply:
done